quickstart/aspnet-core/infra/abbreviations.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Do we need this file? It seems to add complexity when we only use it a couple of times

Reply:
removed this file